lxqt-config-session/man/lxqt-config-session.1

```diff
@@ -1,20 +1,23 @@
-.TH lxqt-config-session "1" "2015-11-05" "LXQt 0.10.0" "LXQt Application/Environment Configuration"
+.TH lxqt-config-session "1" "January 2025" "LXQt 2.1.0" "LXQt Configuration"
 .SH NAME
 \fBlxqt-config-session\fR \- Application settings of \fBLXQt\fR: The Lightweight
 Qt Desktop Environment
 .SH SYNOPSIS
 .B lxqt-config-session
 .br
 .SH DESCRIPTION
-This application handles \fBLXQt\fR default applications settings, autostart
-configuration, and environment settings.
+This application handles \fBLXQt\fR X11 window manager selection, autostart of LXQt modules
+and other applications, settings for Qt scaling, confirmation for session exit,
+screenlocking, besides XDG user directories paths and advanced environment settings.
+.P
+If the \fBlxqt-wayland-session\fR module is installed Wayland-related configuration
+for selecting the compositor and the screenlock command is enabled.
```

Comment:
If the … is installed, (← comma) Wayland-related configurations (← plural) … _will be_ enabled.